cms/templates/group_configurations.html

```diff
@@ -33,37 +31,56 @@ <h1 class="page-header">
+        <div class="wrapper-groups content-groups">
+          <h3 class="title">${_("Content Groups")}</h3>
+            <div class="ui-loading">
+                <p><span class="spin"><i class="icon fa fa-refresh"></i></span> <span class="copy">${_("Loading")}</span></p>
+            </div>
         </div>
+
+      % if should_show_experiment_groups:
+        <div class="wrapper-groups experiment-groups">
+          <h3 class="title">${_("Experiment Group Configurations")}</h3>
+          % if experiment_group_configurations is None:
+            <div class="notice notice-incontext notice-moduledisabled">
+                <p class="copy">
+                    ${_("This module is disabled at the moment.")}
+                </p>
+            </div>
+          % else:
+            <div class="ui-loading">
+                <p><span class="spin"><i class="icon fa fa-refresh"></i></span> <span class="copy">${_("Loading")}</span></p>
+            </div>
+          % endif
+        </div>
+      % endif
       </article>
       <aside class="content-supplementary" role="complementary">
-      <div class="bit">
-        <h3 class="title-3">${_("What can I do on this page?")}</h3>
-        <p>${_("You can create, edit, and delete group configurations.")}</p>
-
-        <p>${_("A group configuration defines how many groups of students are in an experiment. When you create an experiment, you select the group configuration to use.")}</p>
-
-        <p>${_("Click {em_start}New Group Configuration{em_end} to add a new configuration. To edit a configuration, hover over its box and click {em_start}Edit{em_end}.").format(em_start='<strong>', em_end="</strong>")}</p>
-
-        <p>${_("You can delete a group configuration only if it is not in use in an experiment. To delete a configuration, hover over its box and click the delete icon.")}</p>
-
-        <p><a href="${get_online_help_info(online_help_token())['doc_url']}" target="_blank">${_("Learn More")}</a></p>
-      </div>
+        <div class="content-groups-doc">
+          <div class="bit">
+              <h3 class="title-3">${_("Content Groups")}</h3>
+              <p>${_("Use content groups to give groups of students access to specific course content. In addition to course content that is intended for all students, each content group sees content that you specifically designate as visible to them. By associating one or more cohorts to a content group, you can customize the content that a particular cohort or groups sees in your course.")}</p>
+              <p>${_("Click {em_start}New content group{em_end} to add a new content group. To edit the name of a content group, hover over its box and click {em_start}Edit{em_end}. Content groups cannot be deleted.").format(em_start="<strong>", em_end="</strong>")}</p>
+              <p><a href="${get_online_help_info(online_help_token())['doc_url']}" target="_blank">${_("Learn More")}</a></p>
```

Comment:
@cahrens - the `online_help_token` method is used once and statically returns "group_configurations". Unless you object, I'm just going to inline its return value and do the same in the case of the second link.